src/components/NewEmailForm.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import '../css/NewEmailForm.css';
+import { gql, useMutation } from '@apollo/client';
+
+const CREATE_EMAIL = gql`
```

Comment:
I'm glad you pushed to implement this feature -- I think it complements our MVP very well! I'm glad I was able to help you get the GraphQL squared away, too.